components/brave_rewards/browser/android_util.cc

```diff
@@ -5,13 +5,15 @@
 
 #include "base/json/json_reader.h"
 #include "brave/components/brave_rewards/browser/android_util.h"
+#include "brave/common/brave_channel_info.h"
 
 namespace android_util {
 
 ledger::ClientInfoPtr GetAndroidClientInfo() {
   auto info = ledger::ClientInfo::New();
   info->platform = ledger::Platform::ANDROID_R;
   info->os = ledger::OperatingSystem::UNDEFINED;
+  info->channel = brave::GetChannelName();
```

Comment:
when this is fixed to use chrome::GetChannelName the layering violation here needs to be fixed as well. You cannot use brave/common or chrome/common in components

Reply:
cc @aseren FYI for the refactoring we spoke about